.goreleaser.yml

```diff
@@ -0,0 +1,48 @@
+# This is an example .goreleaser.yml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+project_name: zarb-cli
```

Comment:
Right now Zarb binary is  daemon and CLI both. In the next version we might have different binaries for that. Since then could you please rename `zadb-cli` to `zarb`.

Reply:
sure